src/irr/asset/IImageAssetHandlerBase.cpp

```diff
@@ -0,0 +1,9 @@
+#include "irr/asset/IImageAssetHandlerBase.h"
+
+using namespace irr;
+using namespace asset;
+
+IImageAssetHandlerBase::~IImageAssetHandlerBase()
```

Comment:
no cpp file needed, you can inline the constructor

Reply:
it is a destructor as rest

Reply:
just put `{}` witout the `;` after the function declaration in the header (in other words, make the definition inline) you wont need a source file

Reply:
I know  I know, actually it's default